Sprint-3/implement/is-proper-fraction.test.js

```diff
@@ -0,0 +1,14 @@
+const isProperFraction = require("./is-proper-fraction.js")
+
+test("should return false if numerator is greater than the denominator", () => {
+    expect(isProperFraction(5,2)).toBe(false)
+})
+
+test("should return true if numerator is less than the denominator", () => {
+    expect(isProperFraction(3,6)).toBe(true)
+})
+
+test("should return error if denominator is 0;", () => {
+    expect(isProperFraction(5,0)).toBeNull()
```

Comment:
The function is expected to throw an error when the denominator is 0, so you should use .toThrow() instead of .toBeNull().

Reply:
Ok